.release-please-config.json

```diff
@@ -62,11 +62,32 @@
             "changelog-path": "CHANGELOG.md",
             "bump-minor-pre-major": true,
             "bump-patch-for-minor-pre-major": true
+        },
+        "charts/sequencer": {
+            "component": "sequencer-chart",
```

Comment:
I'm wondering if calling it `sequencer` is better? or charts-sequencer since this is a chart first and foremost? WDYT?